bots/starboard/bot/commands/configuration/removebotstars.js

```diff
@@ -0,0 +1,20 @@
+const { Command } = require('../../../../../');
+
+class RemoveBotStarsCommand extends Command {
+	constructor() {
+		super({
+			description: 'Toggles whether or not bot stars are removed.',
+			memberPerms: 'MANAGE_MESSAGES',
+		});
+	}
+
+	async run(msg) {
+		const remove = this.client.settings.get(msg.guild.id, 'removebotreacts', true);
+		if (!msg.member.hasPermission('ADMINISTRATOR') && !msg.owner) return msg.channel.send(`Bot stars are${remove ? ' ' : ' not'} removed.`);
```

Comment:
Could just put a space before this and have `''` or `'not'`, just my personal preference really.

Reply:
I think that's better.